S3 Cleanup (#4475)
awsdocs/aws-doc-sdk-examples: javascriptv3/example_code/s3/README.md

<!--Generated by WRITEME on 2023-02-24 15:05:04.309449 (UTC)-->

> # Amazon S3 code examples for the SDK for JavaScript V3

Reviewer: ```suggestion
# Amazon S3 code examples for the SDK for JavaScript (v3)
```

* [Upload an object to a bucket](actions/put-object.js#L8) (`PutObject`)

### Scenarios

Code examples that show you how to accomplish a specific task by calling multiple
functions within the same service.

* [Create a presigned URL](scenarios/presigned-url-upload.js) 
* [Get started with buckets and objects](scenarios/basic.js) 
* [Upload or download large files](scenarios/multipart-upload.js) 

## Run the examples

### Prerequisites

For prerequisites, see the [README](../../README.md#Prerequisites) in the `javascriptv3` folder.

<!--custom.prerequisites.start-->
<!--custom.prerequisites.end-->

### Instructions

**Note**: All code examples are written in ECMAscript 6 (ES6). For guidelines on converting to CommonJS, see
[JavaScript ES6/CommonJS syntax](https://docs.aws.amazon.com/sdk-for-javascript/v3/developer-guide/sdk-examples-javascript-syntax.html).

**Run a single action**

```bash
node ./actions/<fileName>
```

**Run a scenario**
Most scenarios can be run with the following command:
```bash
node ./scenarios/<fileName>
```

<!--custom.instructions.start-->
Some scenarios are web applications that must be bundled. These scenarios are in the `scenarios/web` folder.

> Follow these steps to run a web scenario. Some scenarios may require extra steps to get them working.

Reviewer: ```suggestion
Follow these steps to run a web scenario. Some scenarios might require extra steps to get them working.
```

> 1. Navigate to the web scenario you would like to run

Reviewer: ```suggestion
1. Navigate to the web scenario you want to run.
```

1. Run `npm i`
1. Run `npm run dev`
<!--custom.instructions.end-->

#### Create a presigned URL

> This example shows you how to create a presigned URL for S3 and upload an object.

Reviewer: ```suggestion
This example shows you how to create a presigned URL for Amazon S3 and upload an object.
```

Author: @lkdavies this one is auto generated by WRITEME. It's pulling from the same metadata that SOS pulls from. Leaving it for now because I think it's accurate.
